registry/sql-registry/registry/database.py

```diff
@@ -47,6 +47,98 @@ def parse_conn_str(s: str) -> Dict:
     }
 
 
+import sqlite3
+
+class SQLiteConnection(DbConnection):
+    @staticmethod
+    def connect(autocommit = True):
+        conn_str = os.environ["CONNECTION_STR"]
+        if "Server=" not in conn_str:
+            logging.debug("`CONNECTION_STR` is not in ADO connection string format")
```

Comment:
maybe better to specify what went wrong. E.g., "CONNECTION_STR does not contain server information."
Also does `conn_str` case-sensitive? If not, you may:
```
if "server=" not in conn_str.lower():
    ...
```